.eslintrc.yml

```diff
@@ -25,6 +25,7 @@ overrides:
       - bin/*
       - lib/cli/**/*.js
       - test/node-unit/**/*.js
+      - test/integration/**/*.js
```

Comment:
would prefer if we can list individual files here; would rather not open the floodgates to ES6 in the integration tests, as there are tentative plans to run some portion of them in a browser

Reply:
https://github.com/mochajs/mocha/pull/3915/commits/563ac00efdfa39da30ec686fcf7a225caf4d4a1e